### PR TITLE
Added logging facilities

### PR DESCRIPTION
**Pull request by GrigoryanArtem**

### CODE REVIEW

File: src/OpenCvSharp/Modules/core/Enum/LogLevel.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+/// <summary>
+/// cv::utils::logging::LogLevel
+/// </summary>
+public enum LogLevel
+{
+    /// <summary>
+    /// for using in setLogVevel() call
+    /// </summary>
+    LOG_LEVEL_SILENT = 0,
```
Comment (shimat) [inline src/OpenCvSharp/Modules/core/Enum/LogLevel.cs]: I suggest removing the common prefix `LOG_LEVEL_` from each field name, which is the same as the type name.

```suggestion
    SILENT = 0,
```

**Review by shimat — COMMENTED** [review]
Thank you!

File: src/OpenCvSharp/Modules/core/Enum/LogLevel.cs (hunk 1 above)
Comment (GrigoryanArtem) [reply src/OpenCvSharp/Modules/core/Enum/LogLevel.cs]: Thanks, that's a good point.

**Comment by shimat** [comment]
CI is not successful but this will be merged. Thank you very much.

**Pull request merged**